src/components/pure/greeting.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Greeting extends Component {
+    
+    constructor(props){
```

Comment:
This is great, but today we dont use this way of declaring a component. With constructor... Check functional components.

https://es.reactjs.org/docs/components-and-props.html